data/json/scenarios.json

```
@@ -538,10 +538,33 @@
     "description": "While being transported to a different military base, the pilot lost control of the helicopter and crashed in the middle of nowhere.  Hopefully some of the soldiers that were with you also survived the accident.",
     "start_name": "Crash site",
     "allowed_locs": [ "sloc_field", "sloc_forest" ],
-    "professions": [ "soldier", "specops", "bio_soldier", "bio_sniper", "bionic_spy", "national_guard", "politician", "mili_pilot" ],
+    "professions": [
+      "soldier",
+      "specops",
+      "bio_soldier",
+      "bio_sniper",
+      "bionic_spy",
+      "national_guard",
+      "hitman",
+      "assassin",
+      "politician",
+      "mili_pilot"
+    ],
     "map_extra": "mx_helicopter",
     "flags": [ "HELI_CRASH", "LONE_START" ]
   },
+  {
+    "type": "scenario",
+    "id": "assassination",
+    "name": "One Last Assassination",
+    "points": 0,
+    "description": "Your mission required you to eliminate a certain high-ranking individual in hiding. Your mark seems to have turned feral while escaping and the world appears to be ending, but not even that can stop you from finishing your mission.",
+    "allowed_locs": [ "sloc_shelter", "sloc_shelter_vandalized" ],
+    "start_name": "Evac Shelter",
+    "professions": [ "assassin", "hitman", "bionic_hitman" ],
```

Adds a few potentially fitting variations.
```suggestion
    "professions": [ "assassin", "hitman", "scoundrel", "gangster", "specops", "bionic_hitman", "bio_gangster", "razorgirl", "bio_sniper" ],
```